python/ray/worker.py

```diff
@@ -1509,7 +1525,7 @@ def init(redis_address=None,
     """
 
     if configure_logging:
-        logging.basicConfig(level=logging_level, format=logging_format)
+        setup_logger(logging_level, logging_format)
```

Comment:
What happens if we call `ray.init(configure_logging=False)`? Will that actually work? What is the use case?

Reply:
Yeah, say when someone wants to set `basicConfig` for example.